shared/js/ui/views/search.es6.js

```diff
@@ -39,7 +39,11 @@ Search.prototype = window.$.extend({},
 
     _handleBurgerClick: function (e) {
       e.preventDefault()
-      this.model.send('burgerClick')
+      if (window.safari && this.model.searchText) {
+          this._handleSubmit(e)
```

Comment:
Could be a bug. For some reason the form submit runs this handler in safari.

Reply:
I wonder if Safari does something weird because it's a `<button>` inside a `<form>`?

Maybe we can just move the button out of the form (https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/2a3b3a33128d34ed044d292fee5a70f77c086104/shared/js/ui/templates/search.es6.js#L13) or try and change it from a `<button>` to a `<a>`.

If something like that works I'd rather do that than leaving the click handler in a hacky place.

Reply:
I didn't get to this one yet. I'll check it out tomorrow.